sale_promotion_rule/data/sale_promotion_rule_data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="product.decimal_discount" model="decimal.precision">
+        <field name="digits">3</field>
```

Comment:
@lmignon why do you change the decimal precision of the discount?
This will change the default value and if the module is installed break existing test in other module